### PR TITLE
use sats internally and btc in UI

### PR DESCRIPTION
**Pull request by hozzjss**
Use the following template to create your pull request

## Description

Sats internally and BTC externally

https://github.com/user-attachments/assets/46e3919b-65b6-4b6f-854d-80f867fd6f59

### CODE REVIEW

File: src/comps/Deposit.tsx [hunk 1]
```diff
@@ -80,14 +80,15 @@ const DepositFlowAmount = ({ setStep, setAmount }: DepositFlowAmountProps) => {
     amount: yup
       .number()
       // dust amount is in sats
-      .min(10_000, `Minimum deposit amount is 10,000 sats`)
-      .max(currentCap, `Current deposit cap is ${currentCap} sats`)
+      .min(10_000 / 1e8, `Minimum deposit amount is ${10_000 / 1e8} BTC`)
```
Comment (aldur) [inline src/comps/Deposit.tsx:83]: Shouldn't this limit come from Emily rather than being hardened as 10_000 sats?

Comment (hozzjss) [reply src/comps/Deposit.tsx:83]: does emily provide a lower limit?

Comment (aldur) [reply src/comps/Deposit.tsx:83]: If it doesn't and you are adding it for UX, it should be at least 100_000 sats

Comment (hozzjss) [reply src/comps/Deposit.tsx:83]: got it!

**Review by aldur — APPROVED** [review]

**Pull request merged**